include/Network/HTTP/HTTPException.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Throwable exceptions to return HTTP status code
+ *
+ * This list of Exception has be extracted from
+ * here http://racksburg.com/choosing-an-http-status-code/
+ */
+namespace Friendica\Network\HTTP;
+
+class HTTPException extends \Exception {
+	var $httpcode = 200;
+	var $httpdesc = "";
+
+	/**
+	 * @brief Base class for HTTP response codes
+	 * @param string $message Error message
```

Comment:
please insert a new line ` *` betwen `@brief` and `@param` (this is not part of the standard but better for readability and we are doing it this way for all other functions)